Project: storybook won't build

**Package**
app-project

**Describe the bug**
Run `yarn storybook` on master in `packages/app-project`.

Webpack errors with
```
Module not found: Error: Can't resolve 'babel-loader' in '/Users/odonnell/zooniverse/front-end-monorepo/packages/app-project'
```

--- a/packages/app-project/package.json
+++ b/packages/app-project/package.json
@@ -66,6 +66,7 @@
     "@storybook/addon-storysource": "~5.3.10",
     "@storybook/addon-viewport": "~5.3.10",
     "@storybook/react": "~5.3.10",
+    "babel-loader": "~8.0.6",
     "babel-plugin-webpack-alias": "~2.1.2",
     "chai": "~4.2.0",
     "dirty-chai": "~2.0.1",

